bpo-3530: Use fix_missing_locations when node transformer adds nodes (#17172)
python/cpython: Doc/library/ast.rst

   statement nodes), the visitor may also return a list of nodes rather than
   just a single node.

   If :class:`NodeTransformer` introduces new nodes (that weren't part of
   original tree) without giving them location information (such as
   :attr:`lineno`), :func:`fix_missing_locations` should be called with

>    new tree to apply location information for compiler.

Reviewer: ```suggestion
   the new sub-tree to recalculate the location information.
```

> 

Reviewer: I suggest to add the example so its clear what needs to be done:

```
tree = ast.parse('foo', mode='eval')
new_tree = fix_missing_locations(RewriteName().visit(tree))
```
